typescript.md

```diff
@@ -30,15 +30,29 @@ defines rules for TypeScript specific features.
 `tsc` has some [options](http://www.typescriptlang.org/docs/handbook/compiler-options.html)
 to make type checks stricter:
 
-* `noImplicitAny`;
-* `noImplicitReturns`;
-* `noUnusedLocals`;
-* `noUnusedParameters`;
-* `noImplicitThis`;
-* `strictNullChecks`.
-
-It's recommended to turn them on as early as possible in development (and migration
-from JS).
+```json
+{
+    "compilerOptions": {
+        "strict": true,
+        "noImplicitReturns": true,
+        "noEmitOnError": true
+    }
+}
+```
+
+It's recommended to turn them on as early as possible in development (and migration from JS).
+
+Also, if you want check unused variables and parameters with `TypeScript` instead `tslint`, you can
```

Comment:
`If you want TypeScript to check for unused variables and parameters instead of tslint...`